src/site/content/en/blog/browser-level-lazy-loading-for-cmss/index.md

```diff
@@ -0,0 +1,283 @@
+It is now on the distributed web ecosystem to make use of the `loading`
+attribute.
+
+Content management systems
+[power about 60% of websites](https://w3techs.com/technologies/overview/content_management),
+so these platforms play a vital role in bringing adoption of modern browser
+features to the web. With a few popular open-source CMSs such as
+[WordPress](https://make.wordpress.org/core/2020/07/14/lazy-loading-images-in-5-5/),
+[Joomla](https://github.com/joomla/joomla-cms/pull/28838), and
+[TYPO3](https://review.typo3.org/c/Packages/TYPO3.CMS/+/63317) having already
+implemented support for the `loading` attribute on images, let's have a look at
+their approaches and the takeaways which are relevant for adopting the feature
+in other CMS platforms as well. Lazy-loading media is a key web performance
+feature that sites should benefit from at a large scale, which is why adopting
+it at the CMS core level is recommended.
+
+## The case for implementing lazy-loading now
+
+### Standardization
+
+Adoption of non-standardized browser features in CMSs facilitates widespread
+testing and can surface potential areas of improvement. However, the general
+consensus across CMSs is that, as long as a browser feature is not standardized,
+it should preferably be implemented in the form of an extension or plugin for
+the respective platform. Only once standardized a feature can be considered for
+adoption in the platform core.
+
+{% Aside 'success' %}
+Browser-level lazy-loading is now part of the WHATWG HTML Standard for
+both
+[`img`](https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-loading)
+and
+[`iframe`](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-loading)
+elements.
+{% endAside %}
+
+### Browser support
+
+Browser support of the feature is a similar concern: The majority of CMS users
+should be able to benefit from the feature. If there is a considerable
+percentage of browsers where the feature is not yet supported, the feature has
+to ensure that it at least has no adverse effect for those.
+
+{% Aside 'success' %}
+[Browser-level lazy-loading is widely supported by browsers](https://caniuse.com/#feat=loading-lazy-attr)
+and
+[the `loading` attribute is simply ignored](/browser-level-image-lazy-loading/#browser-compatibility)
+by those browsers that have not adopted it yet.
+{% endAside %}
+
+### Distance-from-viewport thresholds
+
+A common concern with lazy-loading implementations is that they in principle
+increase the likelihood that an image will not be loaded once it becomes visible
+in the user's viewport because the loading cycle starts at a later stage.
+Contrary to previous JavaScript-based solutions,
+[browsers approach this conservatively](/browser-level-image-lazy-loading/#distance-from-viewport-thresholds)
+and furthermore can fine-tune their approach based on real-world heuristics in
+the field, minimizing the impact, so browser-level lazy-loading should be safe to adopt
```

Comment:
```suggestion
and furthermore can fine-tune their approach based on real-world user experience data,
minimizing the impact, so browser-level lazy-loading should be safe to adopt
```
At first I came here to edit "real-world heuristics in the field" for redundancy (real-world and in the field mean the same thing) but upon looking at this a bit further I think we mean data, not heuristics. Because aren't heuristics rule-of-thumb guidelines that you use to make decisions? Doesn't quite seem like the right word here.

Reply:
Makes sense, "data" is a more accurate term here.